src/components/app/data/hooks/useBFF.test.jsx

```diff
@@ -129,90 +120,50 @@ const mockBFFDashboardData = {
 };
 
 describe('useBFF', () => {
-  const Wrapper = ({ children }) => (
+  const Wrapper = ({ routes = null, children }) => (
     <QueryClientProvider client={queryClient()}>
-      {children}
+      <MemoryRouter initialEntries={[routes]}>
```

Comment:
[clarification] If this `Wrapper` component accepts a `routes` prop, is it expected to accept more than 1 route, given its plurality? Perhaps the `Wrapper` component instead just accepts `initialEntries` as a prop, defaulted to an empty array?